3-Authorization-II/2-call-api-b2c/SPA/src/App.jsx

```diff
@@ -31,17 +31,22 @@ const Pages = () => {
                  * policies may use "acr" instead of "tfp"). To learn more about B2C tokens, visit:
                  * https://docs.microsoft.com/en-us/azure/active-directory-b2c/tokens-overview
                  */
-                let tfp = event.payload.idTokenClaims['tfp'] ?? event.payload.idTokenClaims["acr"];
+                let tfpClaim = event.payload.idTokenClaims['tfp'] ? 'tfp' : 'acr';
                 
-                if (tfp=== b2cPolicies.names.editProfile) {
+                function formatCase(policyName, tfpClaim) {
```

Comment:
Good catch for the case formatting. Let's apply the same logic for the `tfp` claim to avoid issues with string comparisons. If we implement the case formatting in a separate function, please move it to the utils directory so it doesn't come in the way of the auth logic.